web/views/datasets.py

```diff
@@ -78,6 +78,10 @@ def get_form_kwargs(self, step: str = None) -> Dict[str, Any]:
                 Dataset, pk=self.storage.extra_data.get("dataset_id")
             )
             kwargs["dataset"] = dataset
+        elif self.request.user.is_part_of(Groups.DATA_STEWARD.value):
+            # If the user is a data steward, we want to keep the scientific metadata field
+            kwargs["keep_metadata_field"] = True
+
```

Comment:
Should we add user_can_access_scientific_metadata method just in case the group of users changes?

Reply:
Thank you